src/pool/srv_layout.c

```diff
@@ -65,11 +68,70 @@ struct daos_prop_entry pool_prop_entries_default[POOL_PROP_NUM] = {
 		.dpe_val	= DAOS_RECLAIM_SNAPSHOT,
 	}, {
 		.dpe_type	= DAOS_PROP_PO_ACL,
-		.dpe_val_ptr	= NULL,
+		.dpe_val_ptr	= NULL, /* generated dynamically */
 	}
 };
 
 daos_prop_t pool_prop_default = {
 	.dpp_nr		= POOL_PROP_NUM,
 	.dpp_entries	= pool_prop_entries_default,
 };
+
+/**
+ * The default ACL includes ACEs for owner and the assigned group allowing
+ * read/write permissions. All others are denied by default.
+ */
+static struct daos_ace *
+alloc_ace_with_rw_access(enum daos_acl_principal_type type)
```

Comment:
maybe genralize this with a alloc_ace_with_access and pass in the ORed access values? This way we can reuse it later.

Reply:
I left this as-is for now. It's a static method with very limited use, so we can refactor it later if we need to do more with it.